### PR TITLE
Add optional color and style support using paris crate

### PR DESCRIPTION
**Pull request by manio**
More info about formatting:
https://github.com/0x20F/paris

When enabled you can use formatting like this:
info!("I can write <b>bold</b> text or use tags to <red>color it</>");

### CODE REVIEW

**Comment by manio** [comment]
@Drakulix 
Hi Victor, is this ok to merge?

File: Cargo.toml
Comment (Drakulix) [inline Cargo.toml:28]: Please don't depend with wildcard on paris. Crates with wildcard dependencies are denied by crates.io, please just use the latest version.

File: src/lib.rs
Comment (Drakulix) [inline src/lib.rs:43]: Please make the module public, so crates with edition 2018 or newer, can import the macros directly via `use` and not only via `#[macro_use]`.

File: src/paris/mod.rs
Comment (Drakulix) [inline src/paris/mod.rs:1]: Can you maybe just steal the documentation from the [log crate](https://docs.rs/log/0.4.14/log/macro.info.html) and additionally add a link to [paris::formatter::colorize_string](https://docs.rs/paris/1.5.7/paris/formatter/fn.colorize_string.html) so people know where to look for more information?

**Review by Drakulix — CHANGES_REQUESTED** [review]
Sorry, this totally flew under my radar and I forgot to take a look at this. My bad.

This looks mostly fine, just a few remarks here and there, then this is good to go!

Thanks for working on this. :)

**Comment by manio** [comment]
Thank you for review, I'll fix this later according to your comments...

In the mean time please also take a look at my email I've sent you last week, maybe it went to your Spam folder...

File: src/lib.rs
Comment (manio) [reply src/lib.rs]: Not sure If I did this correctly... please take a look at my additional commits.

**Comment by manio** [comment]
Branch updated - I added all changes as new commits for your easier review. I can squash it if you like...

File: src/lib.rs
Comment (Drakulix) [inline src/lib.rs:45]: Oh sorry, this is not what I meant. Exporting the `paris` crate does not expose our new macros.
We need to export our `paris` **module** (now named `paris_macros`). :sweat_smile:

Comment (manio) [reply src/lib.rs:45]: updated ... I hope it is now ok :)

Comment (manio) [reply src/lib.rs:45]: bump :)

**Comment by Drakulix** [comment]
Yeah this looks good now! Thanks.
I am waiting for the CI-checks and once everything is green this can be merged :)

**Comment by manio** [comment]
@Drakulix please re-run CI-checks now...

**Review by Drakulix — APPROVED** [review]

**Comment by manio** [comment]
Thank you :)

**Pull request merged**
